cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -67,10 +68,12 @@
 
     public static final long DEFAULT_FULL_RECONCILIATION_INTERVAL_MS = 120_000;
     public static final long DEFAULT_OPERATION_TIMEOUT_MS = 300_000;
+    public static final long DEFAULT_CONNECT_BUILD_TIMEOUT_MS = 300_000;
```

Comment:
because we already know that the build could take longer and increasing this default doesn't hurt, I would set a higher value compared to the operation timeout (maybe doubling it?).

Reply:
I don't know. I think 5 minutes should be ok in most cases. So I think for the time being we should keep it and change it only if we see a lot of question about this.